cpp/ps0000/p0000-soln-cpp.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`